### ISSUE TITLE
The compare button in the release page does not respond when clicked

### ISSUE THREAD

**Issue by csthenry** [go-gitea/gitea#25705]
### Description

The compare button in the release page does not respond when clicked!

### Gitea Version

1.19.3

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

```
2023/07/05 21:39:12 ...dules/git/command.go:255:Run() [D] [64a57280] /data/git/repositories/henry/hello-git.git: /usr/bin/git -c protocol.version=2 -c credential.helper= -c filter.lfs.required= -c filter.lfs.smudge= -c filter.lfs.clean= rev-parse
2023/07/05 21:39:12 ...dules/git/command.go:255:Run() [D] [64a57280] /data/git/repositories/henry/hello-git.git: /usr/bin/git -c protocol.version=2 -c credential.helper= -c filter.lfs.required= -c filter.lfs.smudge= -c filter.lfs.clean= for-each-ref --format=%(objectname) %(refname) refs/tags/ --sort=-taggerdate
2023/07/05 21:39:12 ...dules/git/command.go:255:Run() [D] [64a57280] /data/git/repositories/henry/hello-git.git: /usr/bin/git -c protocol.version=2 -c credential.helper= -c filter.lfs.required= -c filter.lfs.smudge= -c filter.lfs.clean= cat-file --batch
2023/07/05 21:39:12 ...dules/git/command.go:255:Run() [D] [64a57280] /data/git/repositories/henry/hello-git.git: /usr/bin/git -c protocol.version=2 -c credential.helper= -c filter.lfs.required= -c filter.lfs.smudge= -c filter.lfs.clean= cat-file --batch-check
2023/07/05 21:39:12 ...dules/git/command.go:255:Run() [D] [64a57280] /data/git/repositories/henry/hello-git.git: /usr/bin/git -c protocol.version=2 -c credential.helper= -c filter.lfs.required= -c filter.lfs.smudge= -c filter.lfs.clean= for-each-ref --format=%(objectname) %(refname) refs/heads/ --sort=-committerdate
2023/07/05 21:39:12 ...dules/git/command.go:255:Run() [D] [64a57280] /data/git/repositories/henry/hello-git.git: /usr/bin/git -c protocol.version=2 -c credential.helper= -c filter.lfs.required= -c filter.lfs.smudge= -c filter.lfs.clean= for-each-ref --format=%(objectname) %(refname) refs/tags/ --sort=-taggerdate
2023/07/05 21:39:12 ...dules/git/command.go:255:Run() [D] [64a57280] /data/git/repositories/henry/hello-git.git: /usr/bin/git -c protocol.version=2 -c credential.helper= -c filter.lfs.required= -c filter.lfs.smudge= -c filter.lfs.clean= rev-list --count afcdeccf94fec27f7101bd42b7287f5ba1f7989d
2023/07/05 21:39:12 ...s/context/context.go:222:HTML() [D] [64a57280] Template: repo/release/list
```
I don't see any error messages either in the console or in the gitea log (debug level).

### Screenshots

![{D1BBF612-B092-4d57-B5FF-1FC7B7FE287B}](https://github.com/go-gitea/gitea/assets/32993468/dc72651f-fee6-48f1-aede-4f4d3b0af686)

### Git Version

 2.38.5, Wire Protocol Version 2 Enabled

### Operating System

ubuntu 22.04

### How are you running Gitea?

docker

1.19.3 built with GNU Make 4.3, go1.20.4 : bindata, timetzdata, sqlite, sqlite_unlock_notify

### Database

MySQL

**Comment by csthenry** [comment]
> Do you see error logs in your browser's console?

0 error

**Comment by wxiaoguang** [comment]
Hmm.... a bug .....

Before a fix, I think you can start using 1.20-rc2 or 1.20-nightly since 1.20 is going to be released soon.

**Comment by csthenry** [comment]
> Hmm.... a bug .....
> 
> Before a fix, I think you can start using 1.20-rc2 or 1.20-nightly since 1.20 is going to be released soon.

yes, I don't see any error messages either in the console or in the gitea log (debug level), I suspect this is a bug.

**Comment by wxiaoguang** [comment]
Regression of #24369

**Comment by wxiaoguang** [comment]
->  Fix branch selector on release page in 1.19 #25706

**Comment by wxiaoguang** [comment]
It has been fixed in 1.19 nightly release.
